test/filters/unit_tests.filter

```diff
@@ -616,6 +616,12 @@
 # We disable HATS
 -HatsHandlerTest.HandleSecurityPageHatsRequestPassesArgumentsToHatsService
 
+# These tails fail because we add a new quick action to download bubble to delete local file.
+-DownloadBubbleRowViewInfoTest.InProgressOrCompletedInfo
+-DownloadBubbleRowViewTest.InputProtectorDeniesClicks
+-DownloadBubbleRowViewTest.OnlyEnabledQuickActionsVisible
+-DownloadBubbleRowViewTest.UpdateTimeFromCompletedDownload
```

Comment:
Can we add tests of our own that cover what these tests were covering but with the added support for your changes here?